packages/components/src/notice/index.tsx

```diff
@@ -110,6 +113,11 @@ function Notice( {
 								noDefaultClasses = false,
 								onClick,
 								url,
+							}: NoticeAction & {
+								// `isPrimary` is a legacy prop included for
+								// backcompat, but `variant` should be used
+								// instead.
+								isPrimary?: boolean;
```

Comment:
Should we `Pick` from `ButtonProps`, now that `Button` is typed?

Reply:
👍 Love that idea. Done in b875859cb5.